app/models/wallet.py

```diff
@@ -1,27 +1,29 @@
 from typing import List, Optional
 
-from aries_cloudcontroller.models.indy_cred_info import IndyCredInfo
-from aries_cloudcontroller.models.vc_record import VCRecord
+from aries_cloudcontroller.models.indy_cred_info import (
+    IndyCredInfo as IndyCredInfoAcaPy,
+)
+from aries_cloudcontroller.models.vc_record import VCRecord as VCRecordAcaPy
 from pydantic import BaseModel, Field
 
 
 class SetDidEndpointRequest(BaseModel):
     endpoint: str = Field(...)
 
 
-class ExtendedVCRecord(VCRecord):
+class VCRecord(VCRecordAcaPy):
     credential_id: str = Field(..., alias="record_id")
     record_id: str = Field(..., alias="credential_id")
 
 
 class VCRecordList(BaseModel):
-    results: Optional[List[ExtendedVCRecord]] = None
+    results: Optional[List[VCRecord]] = None
 
 
-class ExtendedIndyCredInfo(IndyCredInfo):
+class IndyCredInfo(IndyCredInfoAcaPy):
     credential_id: str = Field(..., alias="referent")
     referent: str = Field(..., alias="credential_id")
```

Comment:
The double aliasing is a bit confusing to me. e.g. if referent already exists in the AcaPy model, I don't see why we redefine it, aliasing the field which is the alias of the referent ... confusing. Can't we just define credential_id? 

I also think it's worthwhile to add a """description""" to the class detailing why we are masking the acapy model. In this case it's just to rename a field to credential_id